internal/cmd/common.go

```diff
@@ -185,7 +185,7 @@ func getDefaultConfigHome() string {
 	// TODO(adamb): switch to os.UserConfigDir()
 	dir, err := os.UserHomeDir()
 	if err != nil {
-		panic(err)
+		dir = os.TempDir()
```

Comment:
What's the purpose of that? It's pretty odd as each `stateful` command requiring it would start with clean slate.

Reply:
This is because this function makes it so that you have to have a `$HOME` or else the program fails.

Since the gRPC runner has a minimal user environment by default, we can't assume that `$HOME` exists. I figured this is a reasonable approach for now, and maybe we can come up with a better solution later.